node/src/components/block_proposer/deploy_sets.rs

```diff
@@ -1,18 +1,30 @@
-use std::fmt::{self, Display, Formatter};
+use std::{
+    collections::HashMap,
+    fmt::{self, Display, Formatter},
+};
 
 use datasize::DataSize;
 use serde::{Deserialize, Serialize};
 
-use super::{BlockHeight, DeployCollection, FinalizationQueue};
-use crate::{types::Timestamp, Chainspec};
+use super::{BlockHeight, FinalizationQueue};
+use crate::{
+    types::{DeployHash, DeployHeader, Timestamp},
+    Chainspec,
+};
+
+#[derive(Clone, DataSize, Debug, Deserialize, Serialize)]
+pub(super) enum DeployType {
+    Transfer(DeployHeader),
+    Deploy(DeployHeader),
```

Comment:
It's weird if `Deploy` is a `DeployType`. Should it be called `Wasm` or `Code` or something?